src/AppBundle/PullRequests/Repository.php

```diff
@@ -154,7 +156,7 @@ public function removeCommentsIfExists(PullRequest $pullRequest, $pattern, $user
         )
         ;
 
-        if (count($comments) > 0) {
+        if (\count($comments) > 0) {
```

Comment:
Move to use `use function count`

Reply:
I can't do it because of php-cs-fixer rules